roles/openshift_health_checker/test/docker_image_availability_test.py

```diff
@@ -174,6 +178,62 @@ def execute_module(module_name=None, module_args=None, tmp=None, task_vars=None)
         ),
         openshift_deployment_type=deployment_type,
         openshift_image_tag='',
+        group_names=['nodes', 'masters'],
     ))
 
     assert not actual.get("failed", False)
+
+
+@pytest.mark.parametrize("deployment_type, is_containerized, groups, oreg_url, expected", [
+    (
+        "origin", False, ['nodes'], None,
+        set([
+            'openshift/origin-' + infra + ':latest'
+            for infra in NODE_IMAGE_SUFFIXES
+        ])
```

Comment:
I trust your take, perhaps looking at the alternative will be equally bad or worse :)